packages/components/src/DataList/DataList.test.tsx

```diff
@@ -394,6 +394,65 @@ describe("DataList", () => {
       expect(screen.queryByText(mockHeaders.name)).not.toBeInTheDocument();
       expect(screen.queryByText(mockHeaders.email)).not.toBeInTheDocument();
     });
+
+    it("should render the sorting arrows when sorting is specified", () => {
```

Comment:
We do already have this specific test farther down in the file [here](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/DataList/DataList.test.tsx#L471) - but let's delete that one because I like your `it` statement better.

And actually....below that test there is one to `render a title when it's provided` which seems to be tested in the `Header` describe already. We can probably delete that test also

Reply:
Good catch! I didn't notice them. Just deleted them